DDG4/python/DDSim/Helper/ParticleHandler.py

```diff
@@ -153,4 +153,15 @@ def setupUserParticleHandler(self, part, kernel, DDG4):
 
       part.adopt(user)
 
+    elif self.userParticleHandler == "Geant4TVUserParticleHandler":
+      user = DDG4.Action(kernel, "%s/UserParticleHandler" % self.userParticleHandler)
+      try:
+        user.TrackingVolume = DDG4.Kernel().detectorDescription().trackingVolume()
+      # TODO: better error message, suggestions welcome ;)
+      except:
+        logger.error("tracking_volume missing in detector model.")
```

Comment:
```suggestion
      except Exception as e:
        logger.error(f"Failed to check tracking_volume: {e!r}")
```

Reply:
do I still need a try/except now?

Reply:
Well, there could be things that go wrong here, but it will probably crash in any case.